grails-app/jobs/au/org/ala/ecodata/DarwinCoreArchiveJob.groovy

```diff
@@ -0,0 +1,22 @@
+package au.org.ala.ecodata
+
+import grails.core.GrailsApplication
+
+class DarwinCoreArchiveJob {
+
+    GrailsApplication grailsApplication
+    RecordService recordService
+
+    static triggers = {
+        Boolean enabled = grailsApplication.config.getProperty("export.darwinCoreArchive.enabled", Boolean, true)
+        if (enabled) {
+            cron name: "Darwin Core Archive: At 9 PM on Tuesday, Thursday and Saturday",
+                    cronExpression: grailsApplication.config.getProperty("export.darwinCoreArchive.cronSchedule",
+                            "0 0 21 ? * TUE,THU,SAT *")
```

Comment:
Can we please push this back an hour so it's less likely to be interrupted by a release?